terminus-ui/select/src/select.component.spec.ts

```diff
@@ -39,19 +33,17 @@ import {
   getSelectTriggerElement,
   openSelect,
 } from '@terminus/ui/select/testing';
-import { getValidationMessageElement } from '@terminus/ui/validation-messages/testing';
 
-import {
-  TsSelectFormatFn,
-  TsSelectModule,
-} from './select.module';
+import { TsOptionModule } from '@terminus/ui/option';
```

Comment:
this can go with the @terminus imports above (ahead of select), and there should be a line break between @angular and @terminus, around line 12

Reply:
Agree grouping them is preferred. 

Currently we are considering anything outside the current library endpoint to be '3rd party' so we wouldn't require a line break between `@angular` and `@terminus`.